starter-code/index.html

```diff
@@ -41,13 +41,13 @@
 
       <b>$10 cheese pizza</b>
       <ul>
```

Comment:
This a valid way to select specific li's from a list but we have to think about the extreme cases where there may be 10s, if not 100's of list items in a list; in which case adding an ID or unique class is not practical. A better tool to use would be the advanced css selector, nth-child.

Ex: $('.price > ul > li:nth-child(1)').toggle();